COMPATIBILITY.md

```diff
@@ -6,10 +6,11 @@
 The below matrix shows the compatibility of the [`opensearch-java-client`](https://search.maven.org/artifact/org.opensearch.client/opensearch-java) with versions of [`OpenSearch`](https://opensearch.org/downloads.html#opensearch).
 
 | Client Version | OpenSearch Version |
-| --- | --- |
-| 1.0.0 | 1.0.0-1.3.3 |
-| 2.0.0 | 1.3.3-2.0.1 |
-| 2.1.0 | 1.3.3-2.3.0 |
+|----------------|--------------------|
+| 1.0.0          | 1.0.0-1.3.3        |
+| 2.0.0          | 1.3.3-2.0.1        |
+| 2.1.0          | 1.3.3-2.3.0        |
+| 2.2.0          | 1.3.3-2.5.0        |
```

Comment:
What do you think about claiming `1.3.3-2.x` for the latest client version? We are running the integration tests against the [latest unreleased version](https://github.com/opensearch-project/opensearch-java/blob/819cb7cde7cc976bb504e07bad1cbc1629ded26e/.github/workflows/test-integration-unreleased.yml#L19-L20).

Reply:
(client 2.0.0 and 2.1.0 should continue to work against all 2.x server versions as well assuming we do semver correctly)

Reply:
+1

Reply:
@andrross it is a bit far reaching I think, the exact number gives more clarify regarding current OpenSearch releases but I am not sure (we could also put 3.x here since we run against unreleased 3.x as well)

Reply:
@andrross it is a question of being explicit or not. We don't use this syntax in other client repo's so it would be different here. i prefer being explicit as well regarding the versions tested using the integrations tests.

Reply:
@andrross This is a question perhaps to raise in https://github.com/opensearch-project/opensearch-clients because it may be relevant to many clients